test/Login.test.ts

```diff
@@ -99,11 +98,11 @@ it('Can Log in',  async () => {
             algo: Algorithms.Secp256k1,
             encoding: Encoding.HEX,
             type: PubKeyType.SignatureAuthentication2018,
-            value: { tag: "key-owner", publicKey: `${assetKeys.publicKey}` },
-            //value: { tag: "key-owner", publicKey: `${assetKeys.publicKey}` },
+            value: { tag: "key-1", publicKey: `0x${assetKeys.publicKey}` },
         },
```

Comment:
Could you add a test for the publicKey not being having the `0x` prefix? Is that a possible scenario we will encounter?